### PR TITLE
Faster SHK lookup?

### PR DESCRIPTION
**Pull request by joakimmoller**
* reduce number of lookups in shk_level0 table
* slightly different matching results - I think the orignal was wrong. (breakpoints for choosing what value). This choses "closest datapoint before".
* uses som pandas magic

### CODE REVIEW

**Comment by joakimmoller** [comment]
It seem like it did work. But the insert statement in the end is the most time consuming part, I've seen a example taking 12 minutes on my local workstation.

image

File: odincal/odincal/odincal/shk_level1_importer.py [hunk 1]
```diff
@@ -1,124 +1,96 @@
...
-                    pass
-            # query=con.query('''select LO from shk_level1''')
-            # res=query.getresult()
     con.close()
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.DEBUG)
+    psql_connection_string = "postgresql://odin@127.0.0.1/odin?sslmode=verify-ca"
+    shk_level1_importer(14166160221,14166833456, "AC2", psql_connection_string)
```
Comment (skymandr) [inline odincal/odincal/odincal/shk_level1_importer.py]:
```suggestion
    shk_level1_importer(14166160221,14166833456, "AC2", psql_connection_string)

```

**Review by skymandr — APPROVED** [review]
looks good!

Fick du ordning på det här med 0 vs Null?

**Comment by joakimmoller** [comment]
> Fick du ordning på det här med 0 vs Null?

jepp

**Pull request merged**